src/swsssdk/scripts/sonic-db-cli

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+import sys
+import swsssdk
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.NullHandler())
+
+if len(sys.argv) < 3:
+    msg = "More than three arguments are required. 'usage: sonic-db-cli dbname rediscommands'"
```

Comment:
"More than three arguments are required" is not quite an accurate message (more correctly, it requires _at least two arguments_ (the first arg in argv is the program name). I suggest removing that message altogether, and just print the usage message.

Reply:
How about `Usage: sonic-db-cli <db_name> <cmd> [cmd [arg [arg ...]]]`
`redis-cli -h` will give an excellent sample.

Reply:
updated.  @qiluo-msft @jleveque 

 
admin@ASW-7005:~$ sonic-db-cli XXX
'Usage: sonic-db-cli <db_name> <cmd> [cmd [arg [arg ...]]]'. See 'sonic-db-cli -h' for detail example.

admin@ASW-7005:~$ sonic-db-cli -h
Example 1: sonic-db-cli CONFIG_DB keys *
Example 2: sonic-db-cli APPL_DB HGETALL VLAN_TABLE:Vlan10
Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu